### DISCUSSION TITLE
[Improvement] Remember open/closed status of nodes

### DISCUSSION THREAD

**Discussion by dcoales** [vslinko/obsidian-outliner#191] [Archive]
I use both LogSeq and Obsidian.  When you collapse a node in LogSeq it adds a property to that node i.e.

  collapsed:: true

so that it can automatically collapse the node when you next open the app or page.  When you expand the node it deletes the property first (rather than setting it to false) so that effectively you never see the property.

It would be great if this plugin could do the same thing using the same syntax so that 

a) the collapsed state is remembered - which would be great
b) it played nicely with LogSeq.

Thanks
David

**Comment by dcoales** [comment]
Sorry, I can't figure out how to change the label from bug to enhancement :-(

**Comment by vslinko** [comment]
@dcoales Hello, thanks for the idea! I moved it to the other ideas.

**Comment by ozanozyegen** [comment]
This feature is extremely useful if you are taking notes in a hierarchical way. When all the nodes are open, the page looks like a mess. But, by selectively opening nodes, you can easily find what you are looking for.

**Comment by vslinko** [comment]
Should be fixed in https://github.com/vslinko/obsidian-outliner/releases/tag/2.1.1
